core-tests/src/test/java/org/mule/runtime/core/construct/FlowProcessingStrategyTestCase.java

```diff
@@ -39,6 +37,9 @@
   @Mock
   private MuleConfiguration configuration;
 
+  @Mock
+  private SchedulerService schedulerService;
```

Comment:
you could get the scheduler service from the muleContext, there is already one there